examples/resources/sample_input_wordcount

```diff
@@ -1,11 +1,14 @@
-banana
-ski snowboard
-banana ski one two three
-bicycle piano piano three
-two three ski ski jangho
-jangho wonook wonook wonook wonook tennis
-john john jy jy jy wonook
-ski snowboard tennis
-shakespeare sanha girl jy
-piano bicycle tennis
-banana
+wonook m 50
```

Comment:
This file seems like a duplicate with `sample_input_broadcast`

Reply:
It really is. I made it in line with renaming MapReduce example to WordCount, which made `sample_input_mr` into `sample_input_wordcount`. 

BroadcastITCase just happened to use `sample_input_mr` as input, so instead of leaving it to use `sample_input_wordcount`, I made a copy of it and named it `sample_input_broadcast`.

Reply:
Would it not make sense for the broadcast program to use that input, instead of making duplicate files? I feel quite uncomfortable with having duplicate files. [link](https://stackoverflow.com/questions/4226284/how-to-convince-a-colleague-that-code-duplication-is-bad)